src/components/structures/TimelinePanel.js

```diff
@@ -504,12 +504,13 @@ var TimelinePanel = React.createClass({
         // very possible have logged out within that timeframe, so check
         // we still have a client.
         const cli = MatrixClientPeg.get();
-        // if no client or client is guest don't send RR
+        // if no client or client is guest don't send RR or RM
         if (!cli || cli.isGuest()) return;
 
+        let shouldSendRR = true;
+
         var currentReadUpToEventId = this._getCurrentReadReceipt(true);
```

Comment:
it might be worth renaming these as 'read receipt' for consistency with the rest of the function - I'm finding it horribly confusing